docs/pages/benchmarking-profiling.md

```diff
@@ -3,29 +3,29 @@ title: Benchmarking and profiling
 layout: default
 ---
 
-# Benchmarking
+## Benchmarking
```

Comment:
This type of change is because the `title` at the top of the page is essentially `h1`, so these should be `h2`